codux.config.json

```diff
@@ -21,12 +21,12 @@
       "trigger": ["checkout", "pull", "setup"]
     }
   },
-  "styleFilesConfig": {
-    "commonStyleFilePattern": "**/commons.module.scss"
-  },
   "styling": {
     "solution": "scss modules"
   },
+  "styleFilesConfig": {
+    "commonStyleFilePattern": "**/commons.scss"
```

Comment:
I know the name in the google doc is `commons.css`, but the conventional (and grammatically correct) name for this file is `common.css`

Reply:
Wait a min, I'll remove it while we don't need it. It's empty so far